nms/packages/magmalte/app/views/subscriber/SubscriberTable.js

```diff
@@ -416,7 +474,9 @@ function SubscribersTable(props: WithAlert) {
               <Grid item>
                 <SubscriberActionsMenu
                   onClose={() => {
-                    setRefresh(!refresh);
+                    setTimeout(() => {
+                      setRefresh(!refresh);
+                    }, 1500);
```

Comment:
These timeouts are probably going to be re-used across the NMS code. Put it into a constant in a shared space, where we can refer to the same constant from a lot of files.